schulcloud/LoginViewController.swift

```diff
@@ -25,6 +25,10 @@ class LoginViewController: UIViewController {
         usernameTextArea.delegate = self
         passwordTextArea.delegate = self
         usernameTextArea.text = defaults.string(forKey: usernameKey)
+        
+        DispatchQueue.global(qos: .default).sync {
+            recreatePersistentContainer()
+        }
```

Comment:
I chose to put this here because the login screen will be called after logout, as well as on token expiration. I do agree that it would be more elegant to call this on logout rather than on login